dev.krtirtho.Flemozi.yml

```diff
@@ -0,0 +1,31 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+app-id: dev.krtirtho.Flemozi
```

Comment:
while `krtirho.dev` answer, it has an invalid certificate.

Reply:
You had a typo missing a "t"  https://krtirtho.dev. Although I guess this is not the reason

I own the domain but I haven't yet published any site so never needed to certify the primary domain (Not experienced enough to manually certify without help of any service :sweat_smile: )
Btw, there's couple of subdomains I use that are certified .e.g [spotube.krtirtho.dev](https://spotube.krtirtho.dev/) & [fl-query.krtirtho.dev](https://fl-query.krtirtho.dev/)

Reply:
the verification relies on the domain to work.

Reply:
Check now. I've updated certificates & stuff

Reply:
Also there's a present for you
![image](https://github.com/flathub/flathub/assets/61944859/699a5ec2-29c9-431e-abbb-e6432b099043)